### DISCUSSION TITLE
Why cant i copy error message text?

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#2840] [General Questions & Discussion] [answered]
its kind of annoying to have to used a screenshot to keep the error message

**Comment by johnshaughnessy** [comment]
Which error message are you referring to? (Sorry to ask for this, but a screenshot would help here)

**Comment by BonzoMartini** [comment]
Um its *any* error message, not just this one.


regards
Bruno


www.gibsonmartelli.com

**Comment by brianpeiris** [comment, marked as answer]
@BonzoMartini Are you referring to Hubs or Spoke? As John mentioned, it might help if you shared a screenshot of the error you're referring to this one time, so that we can figure out what scenario we need to make the error message copyable.

**Reply by brunomartelli** [reply]
Hi - referring to Spoke. @robin-k-wilson 
![spoke error ](https://user-images.githubusercontent.com/4471896/90070353-a3b20680-dceb-11ea-8be8-eb87680fd9c6.jpg)

**Reply by johnshaughnessy** [reply]
Hi @BonzoMartini - Thanks for following up with a screenshot.

This looks like a bug, as that text _should_ be selectable (as it is for me in this screenshot):
![image](https://user-images.githubusercontent.com/4072106/90171181-f7593a00-dd55-11ea-8311-f0cea1fed0c8.png)
```
Errors loading project:

1 Error:
  Error loading audio :
    Cause:
      Error loading video "https://hubs.local:9090/projects/new":
        Cause:
          DEMUXER_ERROR_COULD_NOT_OPEN: FFmpegDemuxer: open context failed
```

I've created a github issue in the Spoke repository to track this bug. It is here: https://github.com/mozilla/Spoke/issues/1013
Can you please add the following details?
- browser (Firefox, safari, chrome, etc), 
- platform (windows, linux, macos, ios, etc), 
- input device you're using (mouse or trackpad),
- Are there any more error messages in the web console? ( 
Instructions for opening the console in firefox: https://developer.mozilla.org/en-US/docs/Tools/Web_Console#Opening_the_Web_Console 
Instructions for chrome: https://developers.google.com/web/tools/chrome-devtools/open
)

**Reply by brunomartelli** [reply]
How does one fix the video loading error? That is the real problem

**Comment by robinkwilson** [comment]
I'm marking this question as answered for now to get it out of our "todo" pipeline. When you upload a screenshot please @ me so I can open it again.
